index.html

```diff
@@ -72,7 +72,8 @@
     <script type='text/javascript' src='materialize/js/materialize.js'></script>
     <script type="text/javascript" src="d3.v3.js"></script>
     <script type="text/javascript" src="SFMap.js"></script>
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyC-eM4xjHsVZ25wKuIG1PxCQkEs7WTMkeg&callback=initMap" async defer></script>
```

Comment:
This is git just getting confused -- I initally thought I had to add a different geometry library, but it was actually already included. These lines haven't actually changed.